app/src/features/apiClient/screens/PostmanImporterView/components/PostmanImporter/utils.ts

```diff
@@ -178,13 +189,32 @@ const createApiRecord = (
   let contentType: RequestContentType | null = null;
   let requestBody: string | KeyValuePair[] | null = null;
 
+  const handleAddImplicitContentTypeHeader = (contentType: RequestContentType) => {
+    const isContentTypeHeaderSet = headers.find((header: KeyValuePair) => header.key === "Content-Type");
+    if (!isContentTypeHeaderSet) {
+      headers.push({
+        id: headers.length,
+        key: "Content-Type",
+        value: contentType,
+        isEnabled: true,
+      });
+    }
+  };
+
```

Comment:
let's convert this into a pure function and move it out of this function's body as a separate utility